cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/services/ApplicationLifecycleService.java

```diff
@@ -1115,4 +1118,14 @@ public List<ApplicationFilter> getAppFilters(Set<String> artifactNames, @Nullabl
     return builder.build();
   }
 
+  public String decodeUserId(String encodedUserId) {
```

Comment:
check for null

Reply:
I think it will be caught in the try catch?